lib/jsonDir.js

```diff
@@ -32,8 +32,21 @@ var collectJSONS = function (options) {
         }
     }
 
+    function addTimestamp(featureItem) {
+        var timestamp = featureItem["elements"][0]["steps"][0]["output"][0];
```

Comment:
Hi @lkotek 

The Build is failing at ```var timestamp = featureItem["elements"][0]["steps"][0]["output"][0];
                                                                        ^
TypeError: Cannot read property '0' of undefined```

Would you be able to check for undefined and if it's `undefined` then do not add the timeStamp?

Reply:
Hi @gkushang, at first I have to apologize for late answer. I changed it in the way you requested - I am checking for undefined before assignment to timestamp. But I looked at the results of CI and it still fails. Do you have an idea how to resolve it please?